.gitlab/one-pipeline.locked.yaml

```diff
@@ -1,2 +1,4 @@
 include:
-  - remote: https://gitlab-templates.ddbuild.io/libdatadog/one-pipeline/ca/05e116339b9780a138a474d0348e97debfca97f27bbc4ca489cf4e4c90d9cc94/one-pipeline.yml
+  - project: DataDog/apm-reliability/libdatadog-build
+    ref: 0f677257308e1c379af490b754febfb40fa2c06d
```

Comment:
How is this `ref` different from the existing value? 🤔 Can/should we stick to the existing pinned value for now? Where does this value come from?

Reply:
Yup, you are right about bumping the version. I am still trying to figure out the origin of the existing SHA. 

But if the result is green, we could simply adopt the latest SHA I proposed. If not, rollback.